### PR TITLE
Revert: "nginx: Properly redirect non matching domains"

### PR DESCRIPTION
**Pull request by mareksuscak**
For context, see: https://github.com/WeblateOrg/docker/commit/7b8640152caf95b1955813a482719d20a911aea0#r43335920

~~This is a placeholder. I haven't tested it yet due to the lack of time.~~

### CODE REVIEW

File: etc/nginx/default.tpl [hunk 1]
```diff
@@ -40,5 +40,16 @@ server {
 
 server {
     listen 8080 default_server;
+
+    # Health endpoint is exempt from the canonical redirect because it's commonly
+    # being hit from inside the cluster and does not require such level of access.
+    location /healthz/ {
```
Comment (mareksuscak) [inline etc/nginx/default.tpl:46]: Wonder if this should be:

```suggestion
    location ${WEBLATE_URL_PREFIX}/healthz/ {
```

**Comment by nijel** [comment]
Hmm, maybe it would be better to revert 7b8640152caf95b1955813a482719d20a911aea0 instead of adding such exceptions...

**Comment by mareksuscak** [comment]
> Hmm, maybe it would be better to revert [7b86401](https://github.com/WeblateOrg/docker/commit/7b8640152caf95b1955813a482719d20a911aea0) instead of adding such exceptions...

Totally agree.

**Comment by mareksuscak** [comment]
@nijel reverted

**Review by nijel — APPROVED** [review]
Looks good. thanks.

**Comment by mareksuscak** [comment]
Just tested the new `weblate:edge-2020-10-18-c3b561abcae25f4b57edd76cb1d6a14362602382` docker image and I can confirm this commit fixes the issue we were seeing.

**Pull request merged**